ci/stress_tests/application_cluster_template.yaml

```diff
@@ -33,11 +33,8 @@ idle_timeout_minutes: 5
 # Cloud-provider specific configuration.
 provider:
     type: aws
-    region: us-west-2
-    # Availability zone(s), comma-separated, that nodes may be launched in.
-    # Nodes are currently spread between zones by a round-robin approach,
-    # however this implementation detail should not be relied upon.
-    availability_zone: us-west-2b
+    region: us-east-1
+    availability_zone: us-east-1a
```

Comment:
us-west-2 is recommended by AWS for people who live in California and is the region we've standardized around in almost all of our config files, so let's keep that. This also means reverting the AMI IDs.

Reply:
Same with stress_testing_config.yaml

Reply:
This is actually a serious problem for these YAML files. The fact is that with high probability, the user is not going to be in the right region.

This is going to need a long-term solution.

Reply:
We can use something like `aws ec2 describe-images --owners amazon` to get the right image ID I think. Unfortunately out of the box this is quite slow, and requires the right region to be set up in `aws configure` but there should be workarounds to do it. We might be able to make it faster by using some of the flags in https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/finding-an-ami.html or caching the information.

Reply:
How about we switch it to us-east for now, so that the checked in version always reflects the exact configuration that was tested. The next release volunteer can feel free to set it to their personal preference for whatever region they have.

Reply:
True, though eventually these tests will presumably be primarily run in an automated fashion using only a single account, at which point it will be less of an issue.

Reply:
that proposal is fine with me